db_updates/db_transition.py

```diff
@@ -0,0 +1,274 @@
+import sys
+sys.path.append("../brainspell")
+import models
+import updated_models
+import json
+from peewee import *
+import psycopg2
+
+
+def article_transition(start = None):
```

Comment:
Mind adding a docstring to this function? If it's just meant to be run once then thrown away it's fine